dashboard/dashboard-front/src/components/jobs/JobsListView.vue

```diff
@@ -142,6 +172,7 @@ onMounted(() => {
     }
 
     fetchJobs()
+    setupEventStreamClient()
```

Comment:
You might get angry at me after all of this... but... it could be useful for some scenarios (trying to focus on one job at this moment of time to debug/understand what happened, without someone else from pulling rug under feet), to be able to **disable** the auto-update.

Reply:
Do you see it as a debug flag in the code or UI switch?

Reply:
UI switch definitely, for the user to easily change.

Reply:
Alright, I've added a toggle switch for disabling auto-udpate